ui/src/actions/literature.ts

```diff
@@ -324,13 +324,14 @@ export function curateReference({
       await http.post('/literature/reference-self-curation', {
         record_id: recordUuid,
         revision_id: revisionId,
-        reference_index: referenceId,
+        reference_index: referenceId - 1,
```

Comment:
why we need the `-1`?

Reply:
On the backend the reference id is indexed from 0 while on ui from 1, so when I'm sending the reference id from ui it changes the next one in line on the backend

Reply:
isn't a list on UI as well? can we just pass the index of the reference? Are we using something else?

Reply:
The list is on UI, but if an element has an index of 1 on UI on backend it has index 0. That's why when I was passing index straight as it is on UI it was editing the wrong reference.

Reply:
![Screenshot 2023-06-13 at 14 23 40](https://github.com/inspirehep/inspirehep/assets/55505399/f9306011-2908-4048-84d6-8ae954ab0b84)
We get the reference id like this from the backend, it doesn't send us the "real" index which is counted from 0, that's why I need to subtract 1

Reply:
You're right, I instead using label I will use index